x-pack/plugins/enterprise_search/server/routes/app_search/search_settings.test.ts

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { MockRouter, mockRequestHandler, mockDependencies } from '../../__mocks__';
+
+import { registerSearchSettingsRoutes } from './search_settings';
+
+describe('search settings routes', () => {
+  const boosts = { foo: [{}] };
+  const resultFields = { foo: {} };
+  const searchFields = { foo: {} };
+  const searchSettings = {
+    boosts,
+    result_fields: resultFields,
+    search_fields: searchFields,
+  };
```

Comment:
Will we at some point be coming back in and filling this data with example data that matches prod? I find these specific server tests extremely useful for having written examples/documentation of sorts of what kind of data structure we expect from our APIs without having to open up a browser tab and inspect it myself

Reply:
Good point. I can do that now.

Reply:
fd0809ba481d503be220c280f5ccdbedc687d6ff

Reply:
Awesome! That was super helpful for me personally, thank you!